src/internal/hooks/use-base-component/index.ts

```diff
@@ -16,6 +17,7 @@ export interface InternalBaseComponentProps {
  */
 export default function useBaseComponent<T = any>(componentName: string) {
   useTelemetry(componentName);
+  useFocusVisible();
```

Comment:
I think we can safely assume that all our components use focus visible in some way, so let's register it once for all components instead of repeating

Reply:
The Box doesn't. But I agree with the solution nevertheless.